man/homectl.xml

```diff
@@ -544,6 +544,16 @@
         loopback file) the discard logic defaults to on.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--luks-offline-discard=</option><replaceable>BOOL</replaceable></term>
+
+        <listitem><para>Similar to <option>--luks-discard=</option> this controls the trimming of the file
```

Comment:
`<para>Similar to <option>--luks-discard=</option>, controls ...`
(The problem with the version with "this" is that we have a full sentence with a subject and predicate, so we need an adverb, not an adjective, i.e. "similarly" instead of "similar". This all becomes rather awkward, and the version with the comma lets us avoid that.)